### ISSUE TITLE
"Invalid Handler Specified" on docker setup

### ISSUE THREAD

**Issue by Tyler-Hartwig-Pariveda** [vingerha/gtfs2#96]
I am having trouble standing this up as a custom component within a docker hosted home assistant (currently using the stable tag)

To reproduce, I downloaded the latest zip (0.4.8)
After setting this in the right directory, restarting home assistant, and attempting to load the integration, I get this error:

` Config flow could not be loaded: {"message":"Invalid handler specified"} `

Here are the logs from the HA container:
Error Logs:
```
2024-07-01 13:27:54.480 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing platform custom_components.gtfs2.config_flow
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 833, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 850, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/gtfs2/__init__.py", line 12, in <module>
    from .coordinator import GTFSUpdateCoordinator, GTFSLocalStopUpdateCoordinator
  File "/config/custom_components/gtfs2/coordinator.py", line 29, in <module>
    from .gtfs_helper import get_gtfs, get_next_departure, check_datasource_index, create_trip_geojson, check_extracting, get_local_stops_next_departures
  File "/config/custom_components/gtfs2/gtfs_helper.py", line 959
    _LOGGER.debug("Departure time: %s, corrected with delay timestamp: %s", dt_util.parse_datetime(f"{now_date} {row["departure_time"]}").replace(tzinfo=timezone), depart_time_corrected)
                                                                                                                      ^^^^^^^^^^^^^^
SyntaxError: f-string: unmatched '['
2024-07-01 13:27:54.482 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading flow for integration gtfs2: Exception importing custom_components.gtfs2.config_flow
2024-07-01 13:33:09.670 ERROR (MainThread) [homeassistant.loader] Unexpected exception importing platform custom_components.gtfs2.config_flow
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 833, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 850, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/gtfs2/__init__.py", line 12, in <module>
    from .coordinator import GTFSUpdateCoordinator, GTFSLocalStopUpdateCoordinator
  File "/config/custom_components/gtfs2/coordinator.py", line 29, in <module>
    from .gtfs_helper import get_gtfs, get_next_departure, check_datasource_index, create_trip_geojson, check_extracting, get_local_stops_next_departures
  File "/config/custom_components/gtfs2/gtfs_helper.py", line 959
    _LOGGER.debug("Departure time: %s, corrected with delay timestamp: %s", dt_util.parse_datetime(f"{now_date} {row["departure_time"]}").replace(tzinfo=timezone), depart_time_corrected)
                                                                                                                      ^^^^^^^^^^^^^^
SyntaxError: f-string: unmatched '['
2024-07-01 13:33:09.673 ERROR (MainThread) [homeassistant.config_entries] Error occurred loading flow for integration gtfs2: Exception importing custom_components.gtfs2.config_flow
```

**Release used**
- gtfs2 release 0.4.8
- HA type (Container)

**Comment by vingerha** [comment]
I have no clue what you went though but this integration has nothing particualr to do with docker but more with where HA runs on as it runs on the HA env. can you tell me what HA you used before? The integration should have warned with older version of HA so am a bit confused what happened

**Comment by tylerhartwig** [comment]
Sorry used my wrong account before -- I'm not sure which HA version I had before, I installed GTFS2 by copying into my config/custom_components directory, maybe it would have warned through HACS but I installed it manually.

My only issue was not understanding that docker wasn't updating my container/HA version :)

**Comment by vingerha** [comment]
Still no clue, HACS is also only copying so manual is the same, anyhow...it works

**Comment by vingerha** [comment]
resolved now with 0.4.9
